procedures/MT/slew/setup.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<p>Must of use already know that we need DDS Daemon to run these notebooks, but I believe that a reminder or a brief note might help new staff.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>